### DISCUSSION TITLE
Raspberry pi not detecting HERO10 Black

### DISCUSSION THREAD

**Discussion by Lucas-Weaver** [gopro/OpenGoPro#359] [Q&A]
Hey all,

Ive been trying to run the gopro-photo demo to no avail. the demo is able to connect to the camera over bluetooth, read SSID and password, but is unable to connect to the webcam over wifi despite the HERO10 Black network appearing in the raspberry pi's network manager. I have tried both 5ghz and 2.4 ghz wifi signals with no luck. I am also using VNC viewer to control the pi, although im not sure if that is relevant. For reference I am using a raspberry pi 3 b+, with nmcli as the network manager as opposed to the default  dhcpcd. ive attached the log file below, any help is appreciated.

[log.txt](https://github.com/gopro/OpenGoPro/files/11843549/log.txt)

**Comment by shahla-ai** [comment]
@Lucas-Weaver 
watch this [https://www.youtube.com/watch?v=N3acnPbcWPc]()
at time 5:00 there is a note on how to make gopro visible on wifi list
